bin/ch-test

```diff
@@ -1168,6 +1192,24 @@ case $phase in
         test_build
         win
         ;;
+    build-unpack)
```

Comment:
I wonder if rather than making build-unpack a phase, could it be a modifier on build and run? Or, perhaps there's a new phase `unpack`, and `build-unpack` implies `build-images` and `unpack`? Or, maybe `build-images` goes away and is replaced by running `build` with the build-unpack modifier?